Makefile

```diff
@@ -65,7 +65,15 @@ validate-wgsl: $(SNAPSHOTS_OUT)/*.wgsl
 	done
 
 validate-hlsl: $(SNAPSHOTS_OUT)/*.hlsl
-	@set -e && for file in $(SNAPSHOTS_OUT)/*.Compute.hlsl ; do \
+	@set -e && for file in $(SNAPSHOTS_OUT)/*.Vertex.hlsl ; do \
```

Comment:
we discussed this briefly, and we should be able to generate a single HLSL for the module